libsolidity/lsp/LanguageServer.h

```diff
@@ -107,4 +109,22 @@ class LanguageServer
 	Json::Value m_settingsObject;
 };
 
+// Helper base class for implementing handlers.
```

Comment:
Please use our style

Reply:
Note to myself: I think you mean to name it `HandlerBase`.